FrameMetadata/FrameMetadata/TBExampleVideoRender.h

```diff
@@ -0,0 +1,36 @@
+//
```

Comment:
We don't this anymore :)

Reply:
As this sample app is a snapshot of `Custom-Video-Driver`. The sample does use `TBExampleVideoRender` on both publisher and subscriber.

Reply:
You should link to those files instead of snapshotting them